src/main/java/auth/Authentication.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+package auth;
+
+import config.*;
+
+import static config.EnvConfigurationFile.dotenv;
+import static io.restassured.RestAssured.given;
+
+public class Authentication {
+
+    public static Token getAuth() {
+        return
+                given().urlEncodingEnabled(true)
+                        .param(Param.USERNAME,dotenv.get(EnvVariable.USERNAME1.name()))
```

Comment:
Have many static variables for everything does not look good, I think you could use an enum. for example: 
Param.USERNAME and EnvVariable.USERNAME1
Param.PASSWORD and EnvVariable.PASSWORD
and so on